ansible/roles/vm_set/library/vm_topology.py

```diff
@@ -697,9 +697,9 @@ def bind_ovs_ports(self, br_name, dut_iface, injected_iface, vm_iface, disconnec
             VMTopology.cmd("ovs-ofctl add-flow %s table=0,in_port=%s,action=output:%s" % (br_name, dut_iface_id, injected_iface_id))
         else:
             # Add flow from external iface to a VM and a ptf container
-            # filter BGP packets, ICMP packets and layer two packets to EOS,
-            # block other PTF test traffic to EOS for EOS's stability,
-            # PTF can receive all traffic except BGP protocol packets.
+            # Allow BGP, ICMP, SNMP packets and layer2 packets from DUT to neighbors
+            # Block other traffic from DUT to EOS for EOS's stability,
+            # Allow all traffic from DUT to PTF.
             VMTopology.cmd("ovs-ofctl add-flow %s table=0,priority=10,tcp,in_port=%s,tp_src=179,action=output:%s,%s" % (br_name, dut_iface_id, vm_iface_id, injected_iface_id))
```

Comment:
Actually, at first, I didn't allow BGP packets flow into PTF, but  test_bgpmon failed in PR test. It adds a bgp monitor on PTF and verify that DUT is attempting to establish connection to it, that's why I allow BGP flow into PTF.

Reply:
If in the high priority rule, we do not allow specifying injected iface as one of the output, BGP packets will be not be sent to injected interface. They will be filtered out by the high priority flow.
So, for the high priority rules, we have to specify the injected iface as one of the output. For any packets not matching the high priority rules, the will be processed by lower priority rules and only output to injected iface.

Since the whitelist strategy is used here, so multiple rules with different priority can only be used this way.